ui/app/AppLayouts/Profile/views/LanguageView.qml

```diff
@@ -93,6 +95,16 @@ SettingsContentBase {
             Item { Layout.fillWidth: true }
             StatusListPicker {
                 id: languagePicker
+
+                button.interactive: root.languageSelectionEnabled
+                StatusToolTip {
+                    y: parent.height + Style.current.padding
+                    margins: 0
+                    visible: !root.languageSelectionEnabled && languagePicker.button.hovered
+                    orientation: StatusToolTip.Orientation.Bottom
```

Comment:
Can't put the tooltip below... the `z` order is totally messed up here due to bad `StatusListPicker` design